examples/windows/kubernetes-windows-automatic-update.json

```diff
@@ -21,7 +21,7 @@
             "adminPassword": "replacepassword1234$",
             "windowsPublisher": "MicrosoftWindowsServer",
             "windowsOffer": "WindowsServerSemiAnnual",
-            "windowsSku": "Datacenter-Core-1803-with-Containers-smalldisk",
+            "windowsSku": "2019-datacenter-core-smalldisk-2006",
             "enableAutomaticUpdates": false

```

Comment:
This is confusing, the example files is windows-automatic-update but enableAutomaticUpdates is set to false...
I think we should just delete this file as we do not recommend users to use automatic updates.

Reply:
agree